bundle/Resources/public/js/views/plugins/cof-copycontentplugin.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('cof-copycontentplugin', function (Y) {
+    "use strict";
+    /**
+     * Provides the Copy Content plugin.
+     *
+     * @module cof-copycontentplugin
+     */
+    Y.namespace('cof.Plugin');
+
+    /**
+     * Provides plugin for the copy content plugin.
+     *
+     * @namespace cof.Plugin
+     * @class CopyContent
+     * @constructor
+     * @extends eZ.Plugin.CopyContent
+     */
+    Y.cof.Plugin.CopyContent = Y.Base.create('cofCopyContentPlugin', Y.eZ.Plugin.CopyContent, [], {
```

Comment:
I created screencast how it would look like in proposed solution.
![wrong-content-type](https://cloud.githubusercontent.com/assets/12594013/24950044/ef14c64a-1f6f-11e7-9db9-467f0fe9a1c7.gif)